deploy/helm/values.yaml

```diff
@@ -0,0 +1,24 @@
+deploy:
+  kmesh:
+    args:
+    - ./start_kmesh.sh -enable-kmesh -enable-ads=true
+    containerSecurityContext:
+      capabilities:
+        add:
+        - all
+      privileged: true
+    env:
+      bpfLogSize: "12800"
+      meshController: istiod.istio-system.svc:15012
+    image:
+      repository: kmesh
```

Comment:
If it is this value, we cannot install by default

Reply:
Fixed.